sdks/python/apache_beam/io/restriction_trackers.py

```diff
@@ -78,10 +78,11 @@ class OffsetRestrictionTracker(RestrictionTracker):
 
   def __init__(self, start_position, stop_position):
     self._range = OffsetRange(start_position, stop_position)
-    self._current_position = None
+    self._current_position = None  # start_position - 1?
```

Comment:
Looks ok to me to set this to `None`.

Reply:
It complicates the code paths below (and in my subsequent PR). This was a thought to simplify things rather than having to special case None elsewhere. I'll leave it out for now.